clumper/clump.py

```diff
@@ -62,6 +62,33 @@ def read_json(cls, path):
             return Clumper(data)
         return Clumper(json.loads(pathlib.Path(path).read_text()))
 
+    @classmethod
+    def read_jsonl(cls, path, lines=None):
+        """
+        Reads in a jsonl file. You can also specify how many lines you want to read in.
+
+        """
+        data_array = []
+        # Case 1 : Handle local files
+        if path.startswith("https:") and path.startswith("http:"):
+            raise ValueError(
+                "Currently the jsonl files must be in the local machine. Please specify a local file."
+            )
+        else:
```

Comment:
Do you need the `else` statement here?

Reply:
I guess not since I am raising an error. Once I do implement the code to automatically download the file then I might need an else statement but that's just a speculation. For now I will remove it.